service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderControllerAboutToStartTest.java

```diff
@@ -89,9 +88,7 @@ void shouldPopulateUploadRoutingPageConditionVariableWhenRouterIsUpload() {
     }
 
     @Test
-    void shouldPopulateJudgeAndLegalAdvisorInUploadRouteWhenSendNoticeOfProceedingsViaSDOIsToggledOn() {
-        given(featureToggleService.isSendNoticeOfProceedingsFromSdo()).willReturn(true);
-
+    void shouldPopulateJudgeAndLegalAdvisorInUploadRouteWhenSendNoticeOfProceedingsViaSDO() {
```

Comment:
@MockBean
    private FeatureToggleService featureToggleService;

can be removed now, as not needed